src/database/DataTypes/StocktakeBatch.js

```diff
@@ -20,7 +20,8 @@ export class StocktakeBatch extends Realm.Object {
    * @return {boolean} True if |stockOnHand + stocktakebatch.difference| is negative.
    */
   get isReducedBelowMinimum() {
-    const stockOnHand = this.itemBatch.totalQuantity;
+    const { itemBatch } = this;
+    const { totalQuantity: stockOnHand } = itemBatch;
```

Comment:
I would revert this back to original (again as per this destructuring)

Reply:
Reverted.